src/Aspire.Hosting.Azure.Sql/AzureSqlExtensions.cs

```diff
@@ -198,6 +211,61 @@ private static void CreateSqlServer(
         AzureResourceInfrastructure infrastructure,
         IDistributedApplicationBuilder distributedApplicationBuilder,
         IReadOnlyDictionary<string, string> databases)
+    {
+        var sqlServer = CreateSqlServerResourceOnly(infrastructure, distributedApplicationBuilder);
+
+        foreach (var database in databases)
+        {
+            var bicepIdentifier = Infrastructure.NormalizeBicepIdentifier(database.Key);
+            var databaseName = database.Value;
+            var sqlDatabase = new SqlDatabase(bicepIdentifier)
+            {
+                Parent = sqlServer,
+                Name = databaseName,
+            };
+            
+            sqlDatabase.Sku = new SqlSku() { Name = AzureSqlDatabaseResource.FREE_DB_SKU };
```

Comment:
The code that calls into this method is obsolete. I don't think we should change its behavior. We should be deleting the API when we move to a new major version. So let's leave this path using the same behavior as before.

Reply:
So should just I mark it as obsolete and preserve the previous behavior (which means no SKU will be explicitly specified and whatever is the default for Azure SQL at that moment will be used)?